src/AltinnCore/Designer/TypedHttpClients/Exceptions/NotFoundHttpRequestException.cs

```diff
@@ -0,0 +1,19 @@
+using System;
+
+namespace AltinnCore.Designer.TypedHttpClients.Exceptions
+{
+    /// <summary>
+    /// Altinn specific exception which can be caught specifically when a response returns HttpStatusCode.NotFound (404)
+    /// </summary>
+    public class NotFoundHttpRequestException : Exception
+    {
+        /// <summary>
+        /// Constructor
+        /// </summary>
+        /// <param name="message">A custom message for this certain message</param>
```

Comment:
this certain exception or message?

Reply:
Nice find! I rewrote the comment "A custom message for this specific exception"